### PR TITLE
updated to latest librustzcash plus protoc build changes

### PR DESCRIPTION
**Pull request by Oscar-Pepper**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -18,15 +18,16 @@ resolver = "2"
 
 [workspace.dependencies]
 clap = "4.4"
-sapling-crypto = "0.1.0"
-zcash_address = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zingo_backend.0.11.1" }
-zcash_client_backend = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zingo_backend.0.11.1", features = ["lightwalletd-tonic"] }
-zcash_encoding = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zingo_backend.0.11.1" }
-zcash_keys = { git = "https://github.com/zingolabs/librustzcash.git", tag = "zingo_backend.0.11.1", features = ["orchard"] }
+sapling-crypto = "0.1.2"
+zcash_address = { git = "https://github.com/zingolabs/librustzcash.git", rev = "11bcb838d5f11812ee14d21c68b7c5e571176cf4" }
```
Comment (zancas) [inline Cargo.toml]: Why a commit?

Comment (zancas) [reply Cargo.toml]: See suggested `git tag --describe dirty` tag version in PR.

File: zingoconfig/src/lib.rs [hunk 2]
```diff
@@ -393,34 +392,7 @@ impl ZingoConfig {
         log_path.into_boxed_path()
     }
 
-    /// Coin Types are specified in public registries to disambiguate coin variants
```
Comment (zancas) [inline zingoconfig/src/lib.rs]: Let's this deprecate these first, and delete after some grace period.

File: zingolib/src/wallet.rs [hunk 3]
```diff
@@ -267,8 +268,12 @@ impl LightWallet {
             if running_total >= target_amount {
                 break;
             }
-            running_total += Amount::from_u64(D::WalletNote::value_from_note(note.note()))
-                .expect("Note value overflow error");
+            running_total = running_total
```
Comment (zancas) [inline zingolib/src/wallet.rs:271]: LGTM

File: zingoconfig/src/lib.rs (hunk 2 above)
Comment (Oscar-Pepper) [reply zingoconfig/src/lib.rs]: good idea i agree

**Review by zancas — APPROVED** [review]
Nice!

**Pull request merged**